install/kubernetes/helm/istio-init/README.md

```diff
@@ -0,0 +1,58 @@
+# Istio
+
+[Istio](https://istio.io/) is an open platform for providing a uniform way to integrate microservices, manage traffic flow across microservices, enforce policies and aggregate telemetry data.
+
+## Introduction
+
+This chart bootstraps [CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+which are an internal implementation detail of Istio.  CRDs contain all runtime configuration specified by the human
+operator.
+
+This chart must be run to completion prior to running other Istio charts, or other Istio charts will fail to initialize.
```

Comment:
can we build this dependency in other istio charts?   https://github.com/helm/helm/blob/master/docs/helm/helm_dependency.md